test/services/TemplatingService.spec.ts

```diff
@@ -0,0 +1,110 @@
+import Mail from '@/interfaces/Mail';
+import UserRessource from '@/interfaces/UserRessource';
+import TemplatingService from '@/services/TemplatingService';
+import message from '@test/data/message';
+import { AssertionError } from 'assert';
+import { expect } from 'chai';
+import { messaging as firebaseMessaging } from 'firebase-admin';
+import 'mocha';
+
+// Instantiate the service
+const templatingService =
+  new TemplatingService(message.platform, message.template, message.payload, message.languagePayloads);
+const receiver: UserRessource = (message.receivers[0] as UserRessource);
+
+describe('TemplatingService.createMailMessage', () => {
+
+  let mail: Mail;
+
+  beforeEach('create a mail message.', () => {
+    mail = templatingService.createMailMessage(receiver);
+  });
+
+  it('should create a valid e-mail.', () => {
```

Comment:
We should use either `e-mail` or `mail` in the project but not both terms.

Reply:
Changed it to `mail`.